test/e2e/pkg/testnet.go

```diff
@@ -98,6 +100,8 @@ func LoadTestnet(file string) (*Testnet, error) {
 	}
 	dir := strings.TrimSuffix(file, filepath.Ext(file))
 
+	fmt.Printf("Log level: %s\n", manifest.LogLevel)
```

Comment:
Assuming this is just old debug output?

Reply:
Yup I think there was some stuff I wanted to merge and some stuff that was just for debugging and they might have got interlaced